### DISCUSSION TITLE
SO Dataset or Module Dependencies?

### DISCUSSION THREAD

**Discussion by corycandia** [Security-Onion-Solutions/securityonion#5429] [Unsupported Versions]
Does anyone know if there are any special functions being performed on certain datasets or module collections, other than just the logical order of a chaotic log universe?

Do I need to ensure the firewall logs go to the so-firewall index, dataset, and use a specific module?

Right now, I'm just allowing servers, and network devices to forward logs on the default syslog 514, and using grok/dissect and pipelines to handle it rather than dipping into beats, firewall port opening, module configing, and more customizing.

**Comment by dougburks** [comment]
Starting in Security Onion 2.3.60, we support Filebeat modules. If the logs you are sending already have a Filebeat module, then you shouldn't have to write your own grok/dissect or pipelines. 

For more information, please see:
https://blog.securityonion.net/2021/07/security-onion-2360-now-available.html
https://docs.securityonion.net/en/2.3/filebeat.html#modules
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-modules.html
